### ISSUE TITLE
JAWS doesn't pronounce all contents of a textarea

### ISSUE THREAD

**Issue by olegbuchnev** [FreedomScientific/VFO-standards-support#177]
### Summary
JAWS doesn't read all contents of textarea using `insert` + `down arrow` shortcut ("say all")

### Example
https://jsfiddle.net/fy5sc2w1/

Set focus to textarea element and move caret to
1) first line
2) second line
3) third line

### Expected result
I would expect that JAWS pronounces all content of the textarea in all 3 cases

### Actual result
JAWS pronounces
1) this is a test in different lines with different text
2) in different lines with different text
3) with different text

#### JAWS version and build number
JAWS 2018.1811.2 64-bit
JAWS 2019.1901.66 64-bit

#### Browser and version:
Chrome 72 and IE 11

**Comment by jducrot** [comment]
As reported in [issue 119](https://github.com/FreedomScientific/VFO-standards-support/issues/119), this problem has been fixed in JAWS 2020. Test successfully with JAWS 2020.2006.12 ILM in Chrome 84 and FF 79
See [this codepen for more info](https://codepen.io/TPG/pen/NWGBEvv)
